cms/src/main/java/uk/nhs/hee/web/repository/gallery/impl/HEEGalleryWorkflowImpl.properties

```diff
@@ -0,0 +1,2 @@
+#getGalleryTypes
+createGalleryItem=<span class="strong">{0}</span> <span class="good">added</span> "<span class="strong">{1}</span>".
```

Comment:
Do we need this (as I assume `createGalleryItem` property will be inherited from `org/hippoecm/repository/gallery/impl/GalleryWorkflowImpl.properties`) ?